src/components/TimerCounter.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Timer from './Timer'
+
+class TimerCounter extends React.Component  {
+
+  state = {
+        time: 0,
+    }
+  
+
+    getHours() {
+        if (this.state.time >= 3600) {
+            Math.floor((0 % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+		}
+		return 0;
+    }
```

Comment:
Te obliczenia chyba nie są poprawne `0 % ...` dziwnie wygląda.
Nie da się prościej? tj. `return Math.floor(this.state.time / 3600)`?